pgjdbc/src/main/java/org/postgresql/core/PGStream.java

```diff
@@ -643,6 +647,47 @@ public void setMaxResultBuffer(String value) throws PSQLException {
     maxResultBuffer = PGPropertyMaxResultBufferParser.parseProperty(value);
   }
 
+  /**
+   * Method to get MaxResultBuffer from PGStream.
+   *
+   * @return size of MaxResultBuffer
+   */
+  public long getMaxResultBuffer() {
+    return maxResultBuffer;
+  }
+
+  /**
+   * Method to set noticed so far max row size. Set is only performed, when given value is greater
```

Comment:
This comment does not make sense to me. Specifically the first part. "noticed so far max row size" ? As to the second part why do we only want it increasing ?

Reply:
The idea behind this is to keep in maxRowSize the size of biggest read data row. As there may be many data rows send after each other for a query, then value in maxRowSize would contain value noticed so far, because next data rows and their sizes are not read for that moment. We want it increasing, because the size of the biggest among data rows will be used during computing new adaptive fetch size for the query.

Reply:
So this explanation would be good in the comments section.